apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -494,3 +496,125 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
         "weight": 1,
         "feedback": {"correct": (score == 1), "missing": {}, "matches": matches},
     }
+
+
+def test(element_html: str, data: pl.ElementTestData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    name = pl.get_string_attrib(
+        element, "answers-name", defaults.element_defaults["answers-name"]
+    )
+    gradable = pl.get_boolean_attrib(
+        element, "gradable", defaults.element_defaults["gradable"]
+    )
+    if not gradable:
+        return
+
+    # Get raw correct answer
+    a_tru = data["correct_answers"][name]
```

Comment:
Nit, avoiding reading `answers-name` til it's needed:

```suggestion
    gradable = pl.get_boolean_attrib(
        element, "gradable", defaults.element_defaults["gradable"]
    )
    if not gradable:
        return

    # Get raw correct answer
    name = pl.get_string_attrib(
        element, "answers-name", defaults.element_defaults["answers-name"]
    )
    a_tru = data["correct_answers"][name]
```